types/index.d.ts

```diff
@@ -2605,6 +2605,17 @@ export declare namespace Knex {
     forceFreeMigrationsLock(config?: MigratorConfig): Promise<any>;
   }
 
+  interface Seed {
+    seed: (knex: Knex) => PromiseLike<any>;
```

Comment:
```suggestion
    seed: (knex: Knex) => PromiseLike<void>;
```
Probably void, because output ignored